apis/kueue/v1beta1/provisioningrequestconfig_types.go

```diff
@@ -76,6 +82,14 @@ type ProvisioningRequestConfigSpec struct {
 	//
 	// +optional
 	PodSetUpdates *ProvisioningRequestPodSetUpdates `json:"podSetUpdates,omitempty"`
+
+	// mergePodSets specifies the mode for merging PodSets before being passed
+	// to the cluster autoscaler.
+	//
+	// +optional
+	// +kubebuilder:validation:Enum=Disabled;Identical;Equivalent
+	// +kubebuilder:default=disabled
+	MergePodSets MergePodSetsMode `json:"mergePodSets,omitempty"`
```

Comment:
Why not *string and treat nil as Never?

Reply:
It should definitely be ptr as a new field in the schema. 

Should we have "Never" explicit?  There are two schools, I'm ok without it.